frontend/src/app/components/overview/overview.component.html

```diff
@@ -1,5 +1,5 @@
 <app-application-banner id="bannerComponent"></app-application-banner>
-<div id="overview" class="overviewContainer px-sm-4" [ngStyle]="{ 'padding-top.px': overviewPadding | async }">
+<div id="overview" class="overview-container px-sm-4" [ngStyle]="{ 'padding-top.px': overviewPadding | async }">
```

Comment:
why did you change the class name?
Accoring to Google you should use kebab-case for css classes